modules/oe_content_person/oe_content_person.post_update.php

```diff
@@ -116,3 +116,15 @@ function oe_content_person_post_update_30001(): void {
     $updated_form_display->save();
   }
 }
+
+/**
+ * Update the oe_role_name field's label.
+ */
+function oe_content_person_post_update_30002(): void {
```

Comment:
Looks like it is more EWCMS related requirement. Should we do that on component level?

Reply:
It's a corporate content type, I think DGCOMM can decide for components as well and if the field doesn't have the original set label, I don't change it.